packages/lib/src/components/Redirect/components/RedirectShopper/RedirectShopper.tsx

```diff
@@ -1,10 +1,12 @@
 import { Component, h } from 'preact';
+import detectInIframe from '../../../../utils/detectInIframe';
 
 interface RedirectShopperProps {
     beforeRedirect: (resolve, reject, url) => Promise<void>;
     url: string;
     method: 'GET' | 'POST';
     data?: any;
+    redirectFromTop?: boolean;
```

Comment:
I would rename the property, such as `isInsideIframe`, because to me the purpose is not clear from the name `redirectFromTop`.